crates/chain-listener/src/listener.rs

```diff
@@ -53,37 +54,50 @@ pub struct ChainListener {
     listener_config: ChainListenerConfig,
 
     chain_connector: Arc<ChainConnector>,
+    // To subscribe to chain events
     ws_client: WsClient,
+
     ccp_client: Option<CCPRpcHttpClient>,
+
     core_manager: Arc<CoreManager>,
 
     host_id: PeerId,
 
+    // These settings aren't changed
+    // We refresh them on some errors, but it's enough to get them only on start without refreshing
     difficulty: Difficulty,
+    // when the contract was deployed
     init_timestamp: U256,
-    global_nonce: GlobalNonce,
-    current_epoch: U256,
     epoch_duration: U256,
     min_proofs_per_epoch: U256,
     max_proofs_per_epoch: U256,
+    // These settings are changed each epoch
+    global_nonce: GlobalNonce,
+    current_epoch: U256,
 
     proof_counter: BTreeMap<CUID, U256>,
     current_commitment: Option<CommitmentId>,
 
+    // the compute units that are in the commitment
```

Comment:
```suggestion
    // the compute units that are in the commitment and not in deals
```
Because all CUs in deals in CC also, but don't need to produce proofs